doc/man/man3/notcurses_visual.3.md

```diff
@@ -101,11 +101,11 @@ typedef struct ncvgeom {
 
 **int ncvisual_resize_noninterpolative(struct ncvisual* ***n***, int ***rows***, int ***cols***);**
 
-**int ncvisual_polyfill_yx(struct ncvisual* ***n***, int ***y***, int ***x***, uint32_t ***rgba***);**
+**int ncvisual_polyfill_yx(struct ncvisual* ***n***, unsigned ***y***, unsigned ***x***, uint32_t ***rgba***);**
```

Comment:
i think is wrong -- these were taken back to `int`, no? they're coordinates, they ought be.

Reply:
fixed, yeah documentation bug